Makefile.am

```diff
@@ -5253,6 +5253,7 @@ edit_cmd = $(SED) \
         -e 's|@localstatedir[@]|$(localstatedir)|g' \
         -e 's|@runstatedir[@]|$(runstatedir)|g' \
         -e 's|@logpath[@]|$(logpath)|g' \
+        -e 's|@pidpath[@]|$(pidpath)|g' \
```

Comment:
This question has nothing to do with your PR. I'm just trying to understand why it is done this way everywhere.

Why is the second @ between square brackets? Considering these are regular expressions, the character in that position must be present in the set between the brackets. In his case the set is composed of a single @, which make it equivalent to using the @ without the brackets (as was done for the first @). So why using the brackets?

Reply:
It just doesn't work without '[]' (variable isn't substituted). Not sure why though.

Reply:
Surprising. When I try it on the command line, it work without the brackets.

Reply:
The string is actually expanded several times before running.

First the Makefile expansion is done. Autotools also can expand @variable@ in makefile and configure. So I guess this the issue we are facing/workarounding here.